ansible/group_vars/all/openstack

```diff
@@ -48,6 +48,28 @@ openstack_image_elements: []
 
 # List of diskimage-builder element git repos to pull
 openstack_image_git_elements: []
+###############################################################################
+# Configuration variables for a CloudKitty ratings service deployment.
+
+# File path or URL containing Python upper constraints.
+#
+ratings_upper_constraints_file: []
+
+# Environment variables for the OpenStack CLI. By default this is left empty.
+#
+ratings_environment: []
+
+# A list where each item is a dictionary mapping the associated fields, 
+# with the 'mappings' field also being a list of dictionaries. 
+# Example of the mappings and their fields can be found below, 
+# however for more information please refer to the README.md file.
+#
+ratings_hashmap_field_mappings: []
+
+# Much like the field mappings above, the service mappings are a list of
+# dictionaries, however these are not associated with a field.
+#
+ratings_hashmap_service_mappings: []
```

Comment:
Can you rename these with the prefix ``openstack_`` so they match the other roles?

Reply:
By default, these are the actual variable names as used by CloudKitty & OpenStack, and this is the convention used by the other services.